tools/ci_connector_ops/ci_connector_ops/utils.py

```diff
@@ -26,6 +26,7 @@
 DESTINATION_CONNECTOR_PATH_PREFIX = CONNECTOR_PATH_PREFIX + "/destination-"
 ACCEPTANCE_TEST_CONFIG_FILE_NAME = "acceptance-test-config.yml"
 AIRBYTE_DOCKER_REPO = "airbyte"
+GRADLE_PROJECT_RE_PATTERN = r"project\((['\"])(.+?)\1\)"
```

Comment:
Do you care that both `implementation` and `testImplementation` will match?

Reply:
Good question!
I believe that test dependencies should be included at build time because they're problalby use in running java connectors tests. But do change to these tests dependencies should trigger a test pipeline for all the connectors using it? I'm not sure of that, as if I make update to connector acceptance tests I'm not sure I want to run a test on ALL connectors using it. 
I'll differentiate dependencies from test dependencies so that we can control this.